### ISSUE TITLE
Levels

### ISSUE THREAD

**Issue by NathanStanislavsky** [NathanStanislavsky/TetrisWebsite#11]
- [x] Keep track of number of lines cleared and increment
- [x] Add levels to the game depending on number of lines cleared
- [x] Pieces move down faster after each level

**Comment by NathanStanislavsky** [comment]
"In the fixed-goal system, each level requires 10 lines to clear. If the player starts at a later level, the number of lines required is the same as if starting at level 1. An example is when the player starts at level 5, the player will have to clear 50 lines to advance to level 6. In most cases, the level caps at 15 (or a total goal of 150 lines if Endless is toggled off), but the maximum level can go as high as 30. If Endless is toggled on or the line clear goal amount is set higher than 150 lines in games with Level 15 cap, the gravity stops increasing for the remainder until top out or reaching the goal."

TLDR if im implementing an endless version:
- To advance to the next level, the player must clear 10 lines. This requirement remains consistent for each level.
- The game should support a maximum of Level 30 
- Beyond the level cap gravity does not increase

**Comment by NathanStanislavsky** [comment]
# Gravity

- The time for a piece to move down by one row is given by the following formula: Time = (0.8-((Level-1)*0.007))(Level-1)
